src/magicbox/SuggestionsManager.ts

```diff
@@ -189,6 +190,16 @@ export class SuggestionsManager {
     $$(this.magicBoxContainer).setAttribute('aria-expanded', this.hasSuggestions.toString());
   }
 
+  private processActiveSelection(suggestion: HTMLElement, usingKeyboard = false) {
```

Comment:
It would be cleaner to remove the second param and avoid the `if/else` by breaking this method into two:
`processKeyboardSelection()`
`processMouseSelection()`